File: C#/RefactoringWorkshop/Test/Bloaters/DataClumps/CustomerTest.cs
using NUnit.Framework;
using RefactoringWorkshop.Bloaters.Dataclumps;

namespace Test.Bloaters.DataClumps
{
    [TestFixture]
    public class CustomerTest
    {
        [Test]
        public void should_display_customers_address_details() {
            var customer = new Customer
            {
                House = "999",
                Steet = "Some street",
                City = "Gurgaon",
                Postcode = "111888",
                Country = "India"
            };

            Assert.AreEqual("999, Some street, Gurgaon, 111888, India", customer.AddressSummary());
        }

    }
}

File: Java/RefactoringWorkshop/src/test/java/bloaters/dataclumps/CustomerTest.java
package bloaters.dataclumps;

import org.junit.Test;

import static org.junit.Assert.*;

public class CustomerTest {

    @Test public void
    should_display_customers_address_details() {
        Customer customer = new Customer();

        customer.setHouse("999");
        customer.setSteet("Some street");
        customer.setCity("Gurgaon");
        customer.setPostcode("111888");
        customer.setCountry("India");

        assertEquals("999, Some street, Gurgaon, 111888, India", customer.getAddressSummary());
    }
}